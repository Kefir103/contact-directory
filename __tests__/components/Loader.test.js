import React from 'react';
import Loader from '../../src/client/components/Loader';
import { render } from 'enzyme';

describe('<Loader />', () => {
    test('should render correctly', () => {
        const component = render(<Loader />);
        expect(component).toMatchSnapshot();
    });
});
