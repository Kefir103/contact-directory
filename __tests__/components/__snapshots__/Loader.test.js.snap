// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should render correctly 1`] = `
<div
  class="lds-ring"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
